WordPress/gutenberg
# Add interface preferences modal to edit post.

## Description


Replaces the edit post preferences modal with the reusable one from the interface package added in #39153.

## Testing Instructions





1. In the post editor, click the top right hand side "Options" button, and then click the last button on the dropdown, "Preferences". This should open a modal dialog with three tabs of editor preferences. 
2. Check that modal, tabs and preferences all display and work exactly the same as in trunk.

## Screenshots 

<img width="847" alt="Screen Shot 2022-03-03 at 1 45 34 pm" src="https://user-images.githubusercontent.com/8096000/156486391-041d0700-ffd6-4589-87c4-15eae31b5239.png">

## Types of changes


New feature (non-breaking change which adds functionality)


## Checklist:
